blackbox/test_decommission.py

```diff
@@ -343,8 +343,16 @@ def test_graceful_stop_concurrent_queries(self):
             )
             t.start()
 
-        decommission(self.clients[1], self.node_names[1])
-        self.assertEqual(wait_for_cluster_size(self.clients[0], TestGracefulStopDuringQueryExecution.NUM_SERVERS - 1), True)
+        # Ensure we don't hold a connection open to the node getting stopped
+        self.clients[1].close()
+        decommission(self.clients[0], self.node_names[1])
```

Comment:
Why change in the index of the node to be dropped from the cluster 

Shouldn't it be `decommission(self.clients[1], self.node_names[1])`? 
Even if it's by intent, client->name indices should match?

My understanding of this test is that clients[0] is kinda "handler" , we drop node 1 and then again query on node0 that cluster size is 1 less than "full"?

Reply:
The client passed in is used to execute the query `alter cluster decommission ?`, where `?` is `node_names[1]`. 
I wanted to ensure we don't use the client connected to the node we're decomissioning.